python/dgl/udf.py

```diff
@@ -95,33 +94,33 @@ def __len__(self):
         return self.batch_size()
 
     @property
-    def canonical_etype(self):
+    def etype(self):
         """Return the canonical edge type (i.e. triplet of source, edge, and
-        destination node type) for this edge batch, if available."""
-        return self._canonical_etype
+        destination node type) for this edge batch."""
+        return self._etype
```

Comment:
We no longer have `canonical_etype`?

Reply:
The `_etype` field is initialized with canonical etypes.

Reply:
reverted